src/index.js

```diff
@@ -45,8 +45,8 @@ const strikethroughOpeningPatternString = '<s class="slack_strikethrough">'
 const strikethroughClosingPatternString = '</s>'
 const italicOpeningPatternString = '<em class="slack_italics">'
 const italicClosingPatternString = '</em>'
-const blockDivOpeningPatternString = '<div class="slack_block">'
-const blockSpanOpeningPatternString = '<span class="slack_block">'
+const blockquoteOpeningPatternString = '<blockquote>'
+const blockquoteClosingPatternString = '</blockquote>'
```

Comment:
don't remove the classes for backwards comptibility

Reply:
I didn't remove them earlier. but wasn't able to push. as it was thowing an error during commit stating "not used anywhere".

Reply:
Please write the full error message here, not just the second half of it.

Reply:
<img width="714" alt="Screenshot 2024-09-23 at 10 07 34 AM" src="https://github.com/user-attachments/assets/6526b197-fda5-49e4-9507-92c5a9995e19">

Reply:
How is specifying the class name causing this error to be thrown? Can  you try adding the class names again push again?